### PR TITLE
Todo-List Project: Interactive List

### PR DESCRIPTION
**Pull request by peter-akhigbe**
Branch Changes

- Add event listener to the checkbox (`change`).
- Update items object's value for `completed` key to "true" when checkbox is checked.
- Store the updated array of items in local storage
- Implement a function for the "Clear all completed" button

### CODE REVIEW

**Review by jerryowusu — COMMENTED** [review]
Hi @peter-akhigbe 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with no Key Feature section in the readme file. Kindly consider adding it and making sure that all other sections of the microverse readme file is present. The only optional section is the FAQ.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -47,6 +47,16 @@ A simple Todo-List project.
   </ul>
 </details>
```
Comment (BenMukebo) [inline README.md:48]: - [ ] Kindly fix the broken links from your Tech stack section, and I will advise you to add a working link. 👍 If you implement all these suggestions, I'm sure your repo will be more informative and engaging. Thanks. 🙏 

image

File: src/modules/clearAllCompleted.js [hunk 2]
```diff
@@ -8,10 +8,13 @@ const clearAllCompleted = () => {
     if (checkboxes) {
       checkboxes.forEach((checkbox, i) => {
         checkbox.addEventListener('change', (e) => {
```
Comment (BenMukebo) [inline src/modules/clearAllCompleted.js:10]: - [ ] Great job so far 👏🏻 However according to the project requirements, you should create a method related to the status updates (`completed`: `true` / `false`). and make sure the task object's values for `completed` is updated upon user actions. Kindly address this 💯 👍🏻 

- Here is a screenshot with the bo1 and b3 `completed` set to  `true`  but the local storage does not get updated, it's still `false`


image

- Make sure the updated array of items is stored in local storage, so the user gets the correct list values after the page reloads

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @peter-akhigbe,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- A function for the "Clear all completed" button using the `Array.filter()` method is implemented. ✔️ 
- Gitflow was correctly used.  ✔️   
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi @peter-akhigbe  
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * The status updates (completed: true / false).when it marked completed :white_check_mark: 
 * Good Job :+1: :smile: 
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] N/A:wink: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**